Sources/Fluent/Pivot/PivotProtocol.swift

```diff
@@ -59,8 +59,11 @@ extension PivotProtocol where Self: Entity {
         let rightId = try right.assertExists()
 
         try makeQuery()
-            .filter(Left.foreignIdKey, leftId)
-            .filter(Right.foreignIdKey, rightId)
+            .filter(pivotLeftIdKey, leftId)
+            .filter(pivotRightIdKey, rightId)
             .delete()
     }
 }
+
+public var pivotLeftIdKey: String = "left_id"
```

Comment:
What about having the key be something like `left_\(left_id_key)` and `right_\(right_id_key)`